lib/collections/publish.js

```diff
@@ -18,5 +22,9 @@ if (Meteor.isServer) {
 			return TV.find({}, {fields: {user: 0}});
 		}
 	});
-
+	Permissions.allow({
+    	update: function () {
+      		return true;
+		}
+  	});
 };
```

Comment:
Are you updating the Permissions collection client side versus using something like autoform/calling a server side method?